source/_integrations/nanoleaf.markdown

```diff
@@ -50,3 +60,46 @@ name:
 4. The output should include the auth token like *{"auth_token":"xxxxxxxxxxxxxxxxxxxxx"}*, copy the resulting token into your configuration
 
 If you get a 403 Forbidden message, you probably did not press the *ON* button long enough. The time-frame to get a valid token is only 30 seconds, so you have to be quick to issue the curl request.
+
+### Using the Nanoleaf Device
+
+The nanoleaf device is a standard light, so can be used with standard Home Assistant Light services for automations and scripts. For full details see the [Light Integrations Page.](/integrations/light/)
+
+### Getting Useful Values for Automations and Scripts
+
+These examples were written for version 2021.2 with a Nanoleaf Shapes device on version 5.0.0 of firmware, so newer versions may have introduced or changed functionality.
```

Comment:
We don't keep version history in our documentation. It always reflects the latest stable version of Home Assistant.